src/containers/Gurb/pages/Requirements/LightQuestion.jsx

```diff
@@ -0,0 +1,61 @@
+import { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import LightbulbOutlinedIcon from '@mui/icons-material/LightbulbOutlined'
+
+import Chooser from '../../components/Chooser'
+import TextRecomendation from '../../components/TextRecomendation'
+
+import { iconRequirements, iconOffRequirements } from '../../gurbTheme'
+
+import GurbErrorContext from '../../../../context/GurbErrorContext'
+
+const LightQuestion = (props) => {
+  const { values, setFieldValue } = props
+  const { t } = useTranslation()
+  const { setError, setErrorInfo } = useContext(GurbErrorContext)
+
+  const handleLightQuestion = (value) => {
+    setFieldValue('has_light', value)
```

Comment:
[QUESTION] Why a selection is used instead of a message inform to the user that only can continue if has light?

Reply:
It's a design decision. The user has to choose between the 2 options